src/content/docs/apm/agents/go-agent/configuration/distributed-tracing-go-agent.mdx

```diff
@@ -45,7 +45,7 @@ Figure out which services touch your request so you can instrument each of them
 For each service involved in your transactions, you'll perform separate installations of the agent. If some of your services use other languages, simply repeat the [installation steps](/docs/distributed-tracing/enable-configure/quick-start) for those languages.
 
 <Callout variant="tip">
-  The Go agent requires you to manually instrument your Go services, unlike the auto-instrumentation of the other New Relic agents. This means you need to add some lines to your code to use the Go agent.
+  The Go agent requires you to manually instrument your Go services, unlike the auto-instrumentation of the other New Relic agents. This means you need to add some lines to your code to use the Go agent. You'll learn about this when you start the installation below.
```

Comment:
@rhetoric101 Do we want to clarify how or why they'll learn more about this? Is there something in the installation process itself that helps people learn more? Do we just describe more fully later in this doc? I think it might be good to clarify. 

The more I think about this tip, the more skeptical I am that we need it at all. Wouldn't it make more sense to put this with the Go information.

My baseline assumption here is that someone is starting with a Go application that they want to monitor, not that they're trying to decide what language to use to build an app to monitor.

Reply:
@paperclypse the goal here is to just give users a heads-up in case they expect auto-instrumentation. They'll find out soon enough when they start, but I'd like to leave this in.

Reply:
OK, I'll defer to you here